src/main/java/me/zeroeightsix/kami/gui/hudgui/HudElement.kt

```diff
@@ -12,20 +12,22 @@ import me.zeroeightsix.kami.util.graphics.font.FontRenderAdapter
 import me.zeroeightsix.kami.util.math.Vec2d
 import me.zeroeightsix.kami.util.math.Vec2f
 import me.zeroeightsix.kami.util.threads.safeListener
+import me.zeroeightsix.kami.util.translation.TranslationKey
+import me.zeroeightsix.kami.util.translation.TranslationKeyBlank
 import net.minecraftforge.fml.common.gameevent.TickEvent
 import org.kamiblue.commons.interfaces.DisplayEnum
 import org.lwjgl.opengl.GL11.glScalef
 
 open class HudElement(
-    name: String,
     val alias: Array<String> = emptyArray(),
     val category: Category,
-    val description: String,
     val alwaysListening: Boolean = false,
     enabledByDefault: Boolean = false
-) : BasicWindow(name, 20.0f, 20.0f, 100.0f, 50.0f, SettingGroup.HUD_GUI) {
+) : BasicWindow(name = TranslationKeyBlank(),20.0f, 20.0f, 100.0f, 50.0f, SettingGroup.HUD_GUI) {
 
-    val scale by setting("Scale", 1.0f, 0.1f..4.0f, 0.05f)
+    val scale by setting(TranslationKey("gui.hudgui.HudElement.Scale"), 1.0f, 0.1f..4.0f, 0.05f)
+
+    val description: TranslationKey = getTranslationKey("HudElementDescription")
```

Comment:
this has to be a getter otherwise it won't work when changing languages
```
val description get() = TranslationKey("HudElement")
```
would be better imo

Reply:
Ok, thanks.